.github/workflows/node.yaml

```diff
@@ -419,6 +420,7 @@ jobs:
 
             cd packages/${{ matrix.package-name }}
             PACKAGE_NAME=$(yarn info -s . name)
+            PACKAGE_NAME=${PACKAGE_NAME/@sofie-automation/@tv2media}
```

Comment:
We set the PACKAGE_NAME twice? Wouldn't the second override the first?

Reply:
It does. I could use a different variable name in the line above, or squeeze it into a single assignment, but that's potentially one less merge conflict in the future.